tests/layer_tests/tensorflow_tests/test_tf_Conj.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+# Testing operation Conj
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/Conj
+
+class TestComplexConjugate(CommonTFLayerTest):
+
+    def _prepare_input(self, inputs_info):
+
+        rng = np.random.default_rng()
+        assert 'real_part' in inputs_info
+        real_part_shape = inputs_info['real_part']
+        assert 'imag_part' in inputs_info
+        imag_part_shape = inputs_info['imag_part']
+
+        inputs_data = {}
+        inputs_data['real_part'] = 4 * rng.random(real_part_shape).astype(np.complex64) - 2
+        inputs_data['imag_part'] = 4 * rng.random(imag_part_shape).astype(np.complex64) - 2
```

Comment:
Please revert your latest changes because OV real and imag parts should be floating-point and complex tensors appear after `Complex` operation